packages/react/src/themes/teams/components/Alert/alertStyles.ts

```diff
@@ -83,12 +101,42 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertVariables> = {
     flexGrow: 1,
   }),
 
-  action: ({ variables: v }): ICSSInJSStyle => ({
+  action: ({ variables: v, props }): ICSSInJSStyle => ({
```

Comment:
nit: unwritten convention to create `props: p` alias

Reply:
i see that in a lot of places in the code, but i've also heard @levithomason  mention differently. lol. i can update to be consistent though.

Reply:
not sure what @levithomason 's thought's on this one are, hmm.. ok, I don't want to bring more confusion, we can definitely leave it like this